apollo-router/src/plugins/rhai/engine.rs

```diff
@@ -921,6 +1067,133 @@ macro_rules! if_subgraph {
     };
 }
 
+fn http_body_as_bytes(body: Body) -> Result<Bytes, Box<EvalAltResult>> {
```

Comment:
For reviewers...

Rhai doesn't understand `async` so we need a way to enter a `tokio` runtime and execute `async` code which doesn't result in a blocked thread. That's what this little dance is doing.